analyses/molecular-subtyping-EPN/02_ependymoma_generate_all_data.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# Author Teja Koganti (D3B)
+
+import sys
+
+import argparse
+import pandas as pd
+import numpy as np 
+import zipfile
+import statistics
+import  scipy
```

Comment:
I don't think you need both of these, but if you import `stats` from `scipy`, you can use it directly below.
```suggestion
```